conf/messages.cy

```diff
@@ -57,32 +57,55 @@ enrolmentError.paragraph2.text = os ydych yn cael y broblem hon o hyd.
 multiplePTE.true.title = Tax Enrolment Assignment Frontend - Enrolment Present
 multiplePTE.true.heading = This user has no PT enrolment
 multiplePTE.true.text =  The current user does not have an assigned PT enrolment
-enrolledForPTWithSAOnOtherAccount.title = Dim ond eich ID cyfredol all gael mynediad at eich cyfrif treth personol o hyn allan
-enrolledForPTWithSAOnOtherAccount.heading = Dim ond eich ID cyfredol all gael mynediad at eich cyfrif treth personol o hyn allan
-enrolledForPTWithSAOnOtherAccount.paragraph1 = Ar hyn o bryd, rydych wedi mewngofnodi gyda’ch ID ar gyfer Porth y Llywodraeth, sy’n gorffen gyda {0}.
-enrolledForPTWithSAOnOtherAccount.paragraph2 = Dyma’r unig ID a all gael mynediad at eich cyfrif treth personol erbyn hyn. Cadwch fanylion y ID hwn yn ddiogel.
-enrolledForPTWithSAOnOtherAccount.heading2 = Beth sydd wedi digwydd i’m Dynodyddion Defnyddiwr (ID) eraill ar gyfer Porth y Llywodraeth?
-enrolledForPTWithSAOnOtherAccount.h2.paragraph1 = I gael mynediad at Hunanasesiad, bydd yn rhaid i chi fewngofnodi gyda’r ID sy’n gorffen gyda {0}.
-enrolledForPTWithSAOnOtherAccount.h2.paragraph2 = Os oes gennych unrhyw Ddynodyddion Defnyddiwr (ID) eraill ar gyfer Porth y Llywodraeth, byddant yn colli mynediad at eich cyfrif treth personol.
-signInAgain.title = Mae angen i chi fewngofnodi eto gyda’ch ID ar gyfer Hunanasesiad
-signInAgain.heading1 = Mae angen i chi fewngofnodi eto gyda’ch ID ar gyfer Hunanasesiad
-signInAgain.paragraph = Mae hyn er mwyn i chi allu cadw mynediad at Hunanasesiad o’ch cyfrif treth personol.
-signInAgain.heading2 = ID gyda mynediad at Hunanasesiad
-signInAgain.userId = Dynodydd Defnyddiwr (ID)
-signInAgain.email =  E-bost
-signInAgain.lastLoginDate = Mewngofnodwyd ddiwethaf
-signInAgain.phoneNumber = Rhif ffôn
-signInAgain.linkText = Nid yw’r ID uchod yn perthyn i mi
-selfAssessmentInterrupt.title = Rydym yn newid y ffordd yr ydych yn cael mynediad at eich cyfrif treth personol
-selfAssessmentInterrupt.heading = Rydym yn newid y ffordd yr ydych yn cael mynediad at eich cyfrif treth personol
-selfAssessmentInterrupt.paragraph1 = Ar hyn o bryd, mae gennych fwy nag un ID ar gyfer Porth y Llywodraeth sydd â mynediad at eich cyfrif treth personol. Er mwyn diogelu’ch gwybodaeth, rydym yn cyfyngu mynediad i un ID o hyn allan.
-selfAssessmentInterrupt.continue = Yn eich blaen
-keepAccessToSA.title = A ydych am gadw mynediad at Hunanasesiad o’ch cyfrif treth personol?
-keepAccessToSA.heading = A ydych am gadw mynediad at Hunanasesiad o’ch cyfrif treth personol?
-keepAccessToSA.radio.yes = Iawn
-keepAccessToSA.radio.no = Na
-keepAccessToSA.noSA = Nid wyf yn llenwi Ffurflenni Treth Hunanasesiad ar-lein
-keepAccessToSA.error.required = Dewiswch ‘Iawn’ os ydych am gadw mynediad at Hunanasesiad o’ch cyfrif treth personol
+
+
+selfAssessmentInterrupt.title = You currently have more than one Government Gateway user ID
```

Comment:
Needs Welsh translation

Reply:
There are some keys for which the welsh is still not available. If we can review the rest of it and test it rather than blocking the whole ticket. And I will update the code when the welsh is available

Reply:
All the translation is put now